### DISCUSSION TITLE
Clipboard sync customization

### DISCUSSION THREAD

**Discussion by kohend** [input-leap/input-leap#1614] [Feature Requests]
I want to add a feature to allow more fine-grained control over clipboard sharing, especially excluding specific screens from sync, and perhaps an explicit sync option.
Do you think it'll be a useful feature?
Where in the UI should it be? How should it be phrased?

**Comment by shymega** [comment]
I think, given the issue we're seeing with clipboards, a gradual refactor of the clipboard system would be beneficial. 

I would advise a draft PR you can force-push to, and we can review it as work progresses.

I can't advise on UI, as I'm terrible at UI/UX design, and always have been, but I think this is a good idea.

If you can join (& remain) in the IRC/Matrix, maybe @p12tic and I can advise further. Or we can do it here.

**Reply by kohend** [reply]
I can't find the channel through matrix, does it have a different name through the bridge? do I need an invite?

**Reply by shymega** [reply]
@kohend Should be `#inputleap-dev:libera.chat`. I'm using my IRC client right now.
